No need to wrapper script. Systemd guarantees that /usr/ will be mounted before
laptop-mode-tools starts.
diff -Nru laptop-mode-tools_1.61.orig/etc/rules/99-laptop-mode.rules laptop-mode-tools_1.61/etc/rules/99-laptop-mode.rules
--- laptop-mode-tools_1.61.orig/etc/rules/99-laptop-mode.rules	2012-09-06 13:38:13.203845129 -0400
+++ laptop-mode-tools_1.61/etc/rules/99-laptop-mode.rules	2012-09-06 16:05:54.107358474 -0400
@@ -1,3 +1,3 @@
-ACTION=="change", SUBSYSTEM=="power_supply", RUN+="/lib/udev/lmt-udev auto"
-ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="/lib/udev/lmt-udev auto"
-ACTION=="add", SUBSYSTEM=="usb", RUN+="/lib/udev/lmt-udev force modules=usb-autosuspend devices=%k"
+ACTION=="change", SUBSYSTEM=="power_supply", RUN+="/usr/sbin/laptop_mode auto"
+ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="/usr/sbin/laptop_mode auto"
+ACTION=="add", SUBSYSTEM=="usb", RUN+="/usr/sbin/laptop_mode force modules=usb-autosuspend devices=%k"
